__tests__/util.test.ts

```diff
@@ -112,13 +112,15 @@ describe('response merging', () => {
         customResponse: CUSTOM_RESPONSE,
         primaryResponse: GEOCODE_EARTH_RESPONSE
       },
+      true,
```

Comment:
can we get a test that tests this with `false`?

Reply:
I tried but couldn't do it in any way that made sense. This test tests for something completely different. We could add an entire new test case but this is a really weird edge case so I'm tempted to add a TODO here instead?

Reply:
I'd be fine with a TODO.